src/wallet/rpcwallet.cpp

```diff
@@ -2575,6 +2577,258 @@ UniValue fundrawtransaction(const JSONRPCRequest& request)
     return result;
 }
 
+UniValue bumpfee(const JSONRPCRequest& request)
+{
+    if (!EnsureWalletIsAvailable(request.fHelp))
+        return NullUniValue;
+
+    if (request.fHelp || request.params.size() < 2 || request.params.size() > 3)
+        throw runtime_error(
+                            "bumpfee \"txid\" output ( options ) \n"
+                            "\nBumps the fee of a opt-in-RBF transaction.\n"
+                            "This command requires that the transaction with the given txid is in the wallet.\n"
+                            "User specifies which output (e.g., a change output) will be used to bump the fee.\n"
+                            "This command will NOT add new inputs.\n"
+                            "Fee must be high enough to pay a new relay fee.\n"
+                            "If tx has child transactions in mempool, the new fee must pay for them as well.\n"
+                            "This command will fail if fee is not high enough or output is not large enough.\n"
+                            "User can specify totalFee, or use RPC setpaytxfee to set a higher fee rate.\n"
+                            "\nArguments:\n"
+                            "1. \"txid\"              (string, required) The txid to be bumped\n"
+                            "2. output index          (numeric, required) The output index to be decremented\n"
+                            "3. options               (object, optional)\n"
+                            "   {\n"
+                            "     \"confTarget\":       \"n\",          (numeric, optional) Confirmation target (in blocks)\n"
+                            "     \"totalFee\":         \"n\",          (numeric, optional) Total fee to pay, in satoshis (not btc)\n"
```

Comment:
I think, because every else use fee-ratres per KB, we should make clear at this point that `totalFee` and `maxFee` are absolute fee-values (and not per KB).